help/de/Help_ch_Account-Actions.xml

```diff
@@ -1,910 +1,1049 @@
               <para>Bank
               </para>
             </entry>
 
-            <entry>
-              <para>The Bank account type denotes savings or checking accounts held at a bank or other financial
-                institution. Some of these accounts may bear interest. This is also the appropriate
-                account type for check (debit) cards, which directly withdraw payments from a
-                checking account.
+            <entry namest="c2">
+              <para>Die Kontoart Bank bezeichnet Spar- oder Girokonten die bei einer Bank oder einem anderen
+                Finanzinstitut geführt werden. Einige dieser Konten können Zinsen tragen. Diese
+                Kontoart eignet sich auch für Debit- und andere Guthaben-Karten wie z.B.
+                Bonus-Meilen, mit denen Zahlungen direkt einem Girokonto belastet werden.
+                <footnote>
+                  <para>Für viele Konten besteht die Möglichkeit, per Online-Banking Aufträge an das Geldinstitute zu
+                    senden sowie Buchungen abzurufen und zu importieren.
+                  </para>
+                </footnote>
               </para>
             </entry>
           </row>
 
           <row>
-            <entry>
-              <para>Cash
+            <entry namest="c1">
+              <para>Bargeld
               </para>
             </entry>
 
-            <entry>
-              <para>The Cash account type is used to denote the cash that you store in your wallet, shoe box, piggyback,
-                or mattress.
+            <entry namest="c2">
+              <para>Die Kontoart Bargeld bezeichnet das Bargeld, welches Sie z.B. in der Brieftasche, einem Schuhkarton,
+                einer Sparbüchse oder unter Ihrer Matraze aufbewahren. Hier kann auch das Bargeld
+                in der Registrierkasse erfasst werden.
               </para>
             </entry>
           </row>
 
           <row>
-            <entry>
-              <para>Credit Card
+            <entry namest="c1">
+              <para>Kreditkarte
               </para>
             </entry>
 
-            <entry>
-              <para>The Credit Card account type is used to denote credit card accounts, both for cards that allow
-                floating lines of credit (e.g. VISA, MasterCard, or Discover) and with cards that do
-                not permit continuing balances (e.g. American Express)
+            <entry namest="c2">
+              <para>Die Kontoart Kreditkarte kommt für die Konten zu zum Einsatz die, im Gegensatz zu normalen
+                Bankkonten, häufig im Minus sind. Diese Kontoart kann sowohl für Kreditkarten die
+                eine gleitende Kreditlinie erlauben (z. B. VISA, oder MasterCard) als auch bei
+                Karten, die keine laufenden Guthaben zulassen (z. B. American Express), verwendet
+                werden.
               </para>
             </entry>
           </row>
 
           <row>
-            <entry>
```

Comment:
Ich würde den Eintrag Currency komplett löschen. Idealerweise in einem separatren Commit "Drop GnuCash 1 account type currency" in allen Sprachen.

Das gibt doch nur Verwirrung nit den neuen Devisenhandelskonten.